pkg/resources/scheduler/deployment.go

```diff
@@ -66,8 +66,6 @@ func DeploymentCreator(data *resources.TemplateData) reconciling.NamedDeployment
 				"--client-ca-file", "/etc/kubernetes/pki/ca/ca.crt",
 				// We're going to use the https endpoints for scraping the metrics starting from 1.13. Thus we can deactivate the http endpoint
 				"--port", "0",
-				// Force the authentication lookup to succeed, otherwise if it fails all requests will be treated as anonymous and thus fail
```

Comment:
`false` has been the default for a long time, so there's no need to set this. I removed it because it failed anyway because boolean flags do not like the `-flagname true` format anymore.